main/libdrm/APKBUILD

```diff
@@ -23,6 +23,7 @@ build() {
 		--sysconfdir=/etc \
 		--mandir=/usr/share/man \
 		--localstatedir=/var \
+                --enable-freedreno \
```

Comment:
There are spaces instead of tabs. Please use tabs for indentation (and alignment) in APKBUILDs.